### PR TITLE
Store backups in /var/lib/etcd/openshift-backup-$timestamp

### PR DESCRIPTION
**Pull request by sdodson**
Because containerized installs don't mount /var/lib/origin and we
switched to running the backup inside the container that meant that we
were backing up the etcd data into a directory inside the container
filesystem. Since we have no other volume mounted we need to backup into
/var/lib/etcd.

We want to perform the backup inside the container because we cannot be sure that etcd on the host matches the version in the container.

### CODE REVIEW

File: playbooks/common/openshift-cluster/upgrades/etcd/backup.yml [hunk 1]
```diff
@@ -71,12 +62,12 @@
     package:
       name: etcd
       state: latest
-    when: ( embedded_etcd | bool or openshift.common.is_containerized ) and not openshift.common.is_atomic
+    when: embedded_etcd | bool
```
Comment (sdodson) [inline playbooks/common/openshift-cluster/upgrades/etcd/backup.yml]: We no longer rely on host etcd binaries for containerized installs.

File: playbooks/common/openshift-cluster/upgrades/etcd/backup.yml [hunk 2]
```diff
@@ -53,30 +43,22 @@
...
+  - name: Install latest etcd for embedded
     package:
       name: etcd
       state: latest
-    when: ( embedded_etcd | bool or openshift.common.is_containerized ) and not openshift.common.is_atomic
+    when:
+    - embedded_etcd | bool
+    - not openshift.common.is_atomic | bool
 
   - name: Generate etcd backup
```
Comment (ashcrow) [inline playbooks/common/openshift-cluster/upgrades/etcd/backup.yml]: nit: Not related to your PR but this really should be ``Execute etcd backup command``.

File: playbooks/common/openshift-cluster/upgrades/etcd/backup.yml [hunk 3]
```diff
@@ -53,30 +43,22 @@
...
     command: >
       {{ etcdctl_command }} backup --data-dir={{ openshift.etcd.etcd_data_dir }}
-      --backup-dir={{ openshift.common.data_dir }}/etcd-backup-{{ backup_tag | default('') }}{{ timestamp }}
+      --backup-dir={{ etcd_backup_dir }}
```
Comment (ashcrow) [inline playbooks/common/openshift-cluster/upgrades/etcd/backup.yml]: 👍

**Comment by ashcrow** [comment]
aos-ci-test

**Comment by ashcrow** [comment]
@sdodson Bah. The 'out-of-date' warning just popped up.

**Pull request merged**